README.md

```diff
@@ -201,6 +201,16 @@ For further information on usage:
 Azure Active Directory authentication uses temporary authentication tokens to authenticate.
 The `mssql` package does not provide an implementation to obtain tokens: instead, import the `azuread` package and use driver name `azuresql`. This driver uses [azidentity](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#section-readme) to acquire tokens using a variety of credential types.
 
+To reduce friction in local development, `DefaultAzureCredential` can authenticate as the user signed into the Azure CLI.
```

Comment:
`DefaultAzureCredential` isn't the string we use in the connection string.
An app would use either `ActiveDirectoryDefault` or `ActiveDirectoryAzCli` for the `fedauth` connection string parameter.

Reply:
this example probably belongs after line 221

Reply:
I thought about pushing it down to the end but I thought it was important to include it in the overview section to give people a fast path for local development to get started quickly.